src/providers/ldap/ldap_opts.c

```diff
@@ -172,6 +172,7 @@ struct sdap_attr_map rfc2307_user_map[] = {
     { "ldap_user_authorized_service", "authorizedService", SYSDB_AUTHORIZED_SERVICE, NULL },
     { "ldap_user_ad_account_expires", "accountExpires", SYSDB_AD_ACCOUNT_EXPIRES, NULL},
     { "ldap_user_ad_user_account_control", "userAccountControl", SYSDB_AD_USER_ACCOUNT_CONTROL, NULL},
+    { "ldap_user_ad_logon_hours", "logonHours", SYSDB_AD_LOGON_HOURS, NULL},
```

Comment:
The generic RFC2307 and RFC2307bis maps should default the attribute value to NULL. Only the AD LDAP map should set the attribute value to logonHours.